### DISCUSSION TITLE
Add trend to sensor card

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#11276] [Other feature requests]
## The request

It would be cool if we could add the trend to the sensor card. That way we can see how it is changing over time. 

Example:

![image](https://user-images.githubusercontent.com/1444314/86265752-0ce51c80-bb79-11ea-8e3d-445fe0010c5c.png)

**Comment by zsarnett** [comment]
For this, We would just need to get the History of the entity for the specific time that the card specified.

aka. Start and End Time would be the same for the History Fetch. Then Compare that to the current state to get the trend %

The functionality would actually be added to the Entity card as the sensor card extends the entity card.

Boolean to turn this feature on or off and it would replace the icon.

Anything I am missing?

Probably going to hit this one for fun

**Comment by balloob** [comment]
We can use the statistics, that will make it easier. Wonder if we should allow user to set a period ? And if period set, we show it ?
